README.md

```diff
@@ -0,0 +1,40 @@
+[![Build Status](https://travis-ci.org/andela-angene/document-management-system.svg?branch=develop)](https://travis-ci.org/andela-angene/document-management-system)
+[![Coverage Status](https://coveralls.io/repos/github/andela-angene/document-management-system/badge.svg?branch=develop)](https://coveralls.io/github/andela-angene/document-management-system?branch=develop)
+[![Code Climate](https://codeclimate.com/github/andela-angene/document-management-system/badges/gpa.svg)](https://codeclimate.com/github/andela-angene/document-management-system)
+
```

Comment:
Since your app has two sides: API and Clientside, you should probably give detailed information on your readme, detailing the features and endpoints being exposed by your API and also a section that details the features of the clientside.